zebra/interface.c

```diff
@@ -1664,19 +1778,33 @@ static void interface_update_stats(void)
 #endif /* HAVE_NET_RT_IFLIST */
 }
 
+static int parse_ipa_type(const char *ip_version)
+{
+	int ipa_type = IPADDR_NONE;
```

Comment:
This should be of type enum ipaddr_type_t;

This comment applies to ipa_type all over this PR.

Reply:
Thank you for review.
Fixex it.